packages/file_selector/file_selector_windows/pubspec.yaml

```diff
@@ -18,9 +18,11 @@ flutter:
 
 dependencies:
   cross_file: ^0.3.1
+  ffi: ^2.0.1
```

Comment:
dependencies should be ordered, i think ffi should go below

Reply:
`f` comes before `i`, doesn't it?

Reply:
f g h i
i don't know my abcs